src/main/java/com/zsmartsystems/zigbee/sniffer/ZigBeeSniffer.java

```diff
@@ -272,9 +273,11 @@ private static void packetReceived(int sequence, int lqi, int rssi, int[] data)
         WiresharkZepFrame zepFrame = new WiresharkZepFrame();
         zepFrame.setLqi(lqi);
         zepFrame.setChannelId(channelId);
+        zepFrame.setDeviceId((localIeeeAddress.getValue()[1] << 8) + localIeeeAddress.getValue()[0]);
```

Comment:
As briefly discussed in the other issue, I wonder if this is the best approach for the device ID? From a users perspective, this is a "random" number and I wonder if it might be better to set this on the command line so that the user has explicit control over this?

We basically have the two options -:

* Use the IEEE address. If there are multiple sniffers in the network, then the user needs to make a note of the IEEE address and then try and correlate this "large number" in Wireshark which may not be quite so obvious.
* Use a value defined on the command line. Here the user has the ability to control the ID and can use their own numbering based on their own thinking. Numbering could be based on channels, or it could be based on location in a building - floors, rooms in a hotel etc. If we use the "rooms in a hotel" as an example, with the IEEE address you need to know what IEEE address is linked to what floor, or room - to me it seems easier and better to simply set it explicitly.

Reply:
I'll check with @mikomarrache what he can do, if adding a new parameter is possible and having of default value of 1 if no parameter is specified.

Reply:
I added a command line option that allows specifying a device id to be included in the ZEP frames. If not specified, the last 16 bits of the EUID is used.